aiida/restapi/translator/base.py

```diff
@@ -333,6 +255,8 @@ def def_order(columns):
             return order_dict
 
         ## Assign orderby field query_help
+        if 'id' not in orders[self._result_type] and '-id' not in orders[self._result_type]:
```

Comment:
Does it mean that if the user specifies an order, anyway 'id' is added as a secondary order? (It's probably ok, just double checking)